pkg/leclient/utils.go

```diff
@@ -36,3 +37,20 @@ func GetSecret(kubeClient client.Client, secretName, namespace string) (*corev1.
 
 	return s, nil
 }
+
+// IsCertificateIssuerLE takes an issuer name on a certificate and determines if it's a Let's Encrypt CA
+func IsCertificateIssuerLE(issuer pkix.Name) bool {
+	if len(issuer.Organization) > 0 {
+		for _, o := range issuer.Organization {
+			if o == "Let's Encrypt" {
```

Comment:
Can we make these constants?

Reply:
If we aren't reusing them, what's the benefit of a constant?

Reply:
I want the tests to be hard coded, as I don't want them touched if we mess with this function (the actual certs won't ever change). The tests also have more data then we actually check in this function.

So I'm intentionally *not* reusing strings with the tests.. and if not there, then we don't really reuse these strings anywhere (so the benefit of allocating a constant isn't there).

Reply:
Fair enough, thanks.